tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/ReleasePlan/SpecWorkFlowTool.cs

```diff
@@ -372,10 +372,6 @@ public async Task<string> GetSDKPullRequestDetails(string language, int workItem
                 }
 
                 var pr = await devopsService.GetSDKPullRequestFromPipelineRunAsync(buildId, language, workItemId);
-                var parsedLink = DevOpsService.ParseSDKPullRequestUrl(pr);
-
-                var rp = await devopsService.GetReleasePlanForWorkItemAsync(workItemId);
-                await UpdateSdkPullRequestDescription(parsedLink, rp);
```

Comment:
Let's remove these two lines of code that update description. We still need the MCP tool to find the SDK pull request from build if user runs SDK generation without a release plan.